docker/k8s/pmm-client/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM vitess/base AS base
+
+FROM debian:stretch-slim
+
+# Copy CA certs for https calls
+COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+RUN apt-get update && \
+    apt-get upgrade -qq && \
+    apt-get install wget -qq --no-install-recommends
+
+RUN wget https://www.percona.com/redir/downloads/pmm-client/1.6.1/binary/debian/stretch/x86_64/pmm-client_1.6.1-1.stretch_amd64.deb
```

Comment:
Do they not have an apt repo we can add?

Reply:
They do. This seemed easier, but I can switch to their apt setup